functions.js

```diff
@@ -1 +1,71 @@
 // YOUR CODE HERE
+//maxOrMin(num1, num2, max)
+function maxOrMin(num1, num2, max) {
+  if (max) {
+    if (num1 > num2) {
+      return num1;
+    } else {
+      return num2;
+    }
+  } else {
+    if (num1 > num2) {
+      return num2;
+    } else {
+      return num1;
+    }
+  }
+}
+
+//sillySum(arr)
+function sillySum(arr) {
+  for (i = 0; i < arr.length; i++) {
+    console.log(arr[i] * i);
+  }
+}
+
```

Comment:
You got the gist, but the instructions ask for the sum of the values in the array multiplied by the index.  Should return one number.